_schemaV2.graphql

```diff
@@ -3549,6 +3549,25 @@ type CitySponsoredContent {
   ): ShowConnection
 }
 
+# A collection of artworks
+type Collection {
+  # True if this is the default collection for this user, i.e. the default Saved Artwork collection.
+  default: Boolean!
```

Comment:
This is _almost_ always true, and serves a reminder that we should probably proceed with some [legacy data cleanup](https://www.notion.so/artsy/Wishlists-5b45360488f540f79212c0dd00d66466#a8b0090795a1482aa4560df99b345d65) to ensure that this is in fact always true.